config/default/core.entity_view_display.node.hs_publications.hs_vertical_card.yml

```diff
@@ -54,14 +56,19 @@ bundle: hs_publications
 mode: hs_vertical_card
 content:
   field_hs_publication_author:
-    type: entity_reference_entity_view
+    type: entity_reference_label
```

Comment:
would we not want the user to be able to click on the author's name to view all publications by that author?

Reply:
Good question. I don't think so, not in this case, as it wasn't scoped or mocked. If we did do it (here or in the future), we'd likely want to be sure we did it everywhere, not just on the homepage card. Let's bring this up as a future enhancement later, as it hasn't been asked for.